### PR TITLE
Auto update collector and scanner

### PR DESCRIPTION
**Pull request by janisz**
This PR will enable weekly updates of collector and scanner version.

See:
- https://github.com/stackrox/stackrox/pull/4158
- https://github.com/stackrox/stackrox/pull/4156

### CODE REVIEW

File: .github/workflows/update_collector_and_scanner.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        signoff: false
+        delete-branch: true
+        title: Update COLLECTOR_VERSION
+        body: |
+          Update COLLECTOR_VERSION to latest master version
+        labels: |
+          ci-all-qa-tests
+          dependencies
+        assignees: janisz
+        reviewers: RTann
```
Comment (Molter73) [inline .github/workflows/update_collector_and_scanner.yaml:40]: Think we can probably remove Ross as a reviewer here.

File: .github/workflows/update_collector_and_scanner.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        branch: update_collector_version
+        signoff: false
+        delete-branch: true
+        title: Update COLLECTOR_VERSION
+        body: |
+          Update COLLECTOR_VERSION to latest master version
+        labels: |
+          ci-all-qa-tests
+          dependencies
+        assignees: janisz
```
Comment (Molter73) [inline .github/workflows/update_collector_and_scanner.yaml:39]: Do we need an assignee? I think these PRs will be mostly on the collector team to review and get merged.

**Review by Molter73 — APPROVED** [review]
/lgtm

At least for the collector side of things

File: .github/workflows/update_collector_and_scanner.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        title: Update SCANNER_VERSION
+        body: |
+          Update SCANNER_VERSION to latest master version
+        labels: |
+          ci-all-qa-tests
+          dependencies
+        assignees: janisz
+        reviewers: RTann
+        team-reviewers: |
+          backend-dep-updaters
```
Comment (msugakov) [inline .github/workflows/update_collector_and_scanner.yaml:87]: I just came here to write this comment: please add scanner or data shepherds team as reviewers.

Comment (janisz) [reply .github/workflows/update_collector_and_scanner.yaml]: There is no scanner team so I added devs from `@scanner-team` slack group and data shepherds.

**Pull request merged**